upstairs/src/downstairs.rs

```diff
@@ -1848,30 +1848,36 @@ impl Downstairs {
             extent_id: ExtentId(0),
             block: BlockOffset(0),
         });
-        let awrite = if is_write_unwritten {
-            IOop::WriteUnwritten {
-                dependencies,
-                start_eid: start.extent_id,
-                start_offset: start.block,
-                data: write.data.freeze(),
-                blocks: write.blocks,
-            }
+        let (awrite, acked) = if is_write_unwritten {
```

Comment:
> Opening as a draft for now, because we should decide whether we want to keep the difference in behavior between Write (which does fast-ack) and WriteUnwritten (which does not).

I don't remember why we decided this, and without knowing that I say we remove this difference and have both fast ack. WriteUnwritten may even do no work at all, meaning whatever overhead we can get rid of would be a win.

Reply:
Okay, done in 75c8c5e4dab28a02297dab9246185fb8e53eab43